src/main/webapp/js/user-page-loader.js

```diff
@@ -49,6 +49,11 @@ function showMessageFormIfViewingSelf() {
 /** Fetches messages and add them to the page. */
 function fetchMessages() {
   const url = '/messages?user=' + parameterUsername;
+  const parameterLanguage = urlParams.get('language');
+  let url = '/messages?user=' + parameterUsername;
```

Comment:
is there a reason you are setting this twice?

Reply:
i got rid of the redundancy